### ISSUE TITLE
STEAM: Favorites keep disappearing

### ISSUE THREAD

**Issue by longboardtard** [ValveSoftware/steam-for-linux#4577]
#### Your system information
- Steam client version: `steam 1.0.0.52-2`
- Distribution (e.g. Ubuntu): `ArchLinux`
- Opted into Steam client beta?: No
- Have you checked for system updates?: Yes
#### Please describe your issue in as much detail as possible:

This is a long lasting bug, I can't believe it still exists. I've been a long time user of `steam` and have had my share of this bug already. It's happened across many `steam` and system updates. The first time it happened to me was in Windows, but now that I use Linux almost exclusively, and have been using it for some years now, I'm just getting tired of it happening here too. The description of the problem is simple:

I add some servers to my favorites, and everything is fine, they are there, but then if I happen to go into offline mode, when I come back into online mode, my favorite servers disappear. It doesn't always happen, meaning I can't get it to happen consistently 100% of the time, but I can get it to happen 75% of the time which is still significantly annoying. Sometimes I believe it's just happened randomly instead, even without going offline/online.
#### Steps for reproducing this issue:
1. Add servers to your favorites.
2. Go offline, then online again.
3. Enjoy.

**Comment by itzexor** [comment]
Same issue here, except I don't use offline mode. It used to only occur once in a while and be resolved by a steam restart, but lately my favorites have been disappearing completely or just showing up as "No internet games responded to the query" every time. This is really annoying.

**Comment by gdrewb-valve** [comment]
#3511 reports something very similar on Windows.

**Comment by gdrewb-valve** [comment]
Note that this forum is for Linux-specific bugs and since this appears to happen on Windows also this is not the ideal place to report it.

**Comment by longboardtard** [comment]
Please, read my description again. This **is** happening on Linux. I only mentioned the fact that it's occured to me in Windows as well just to point out that this isn't Linux specific, it's a general problem, and it's been there for a long long time.

**Comment by gdrewb-valve** [comment]
Right, this forum is for problems that only happen on Linux/Mac (this forum is for the porting team).  Things that happen on all platforms are passed over to the regular Steam team.

**Comment by kisak-valve** [comment]
Hello @longboardtard, @itzexor, have either of you seen this issue on an up to date system?

**Comment by kisak-valve** [comment]
Closing pending feedback.

**Comment by rpr1mera** [comment]
Hello, this happened to me again. At the time it happened it was a fully updated steam package (Archlinux).

steam 1.0.0.61-1

**Comment by longboardtard** [comment]
This happened to me again a couple of weeks back, sorry I hadn't replied. At the time it happened it was the latest steam package available in archlinux:

steam 1.0.0.61-1
